python/pickle.load.yml

```diff
@@ -0,0 +1,9 @@
+rules:
```

Comment:
We'd rather not include a rule for `pickle.load` as this rule already exists within the official Semgrep security audit ruleset, and we don't want to duplicate rules in that one.